tagstudio/src/qt/helpers/file_opener.py

```diff
@@ -134,6 +139,64 @@ def setFilePath(self, filepath):
         """
         self.filepath = filepath
 
+    def truncate_single_filepath(self, filepath):
```

Comment:
please add some tests for this method, thanks.

Reply:
There currently doesn't seem to be tests for the front end, at least until 332 is merged

Reply:
this function can be tested even without QT, no? `Path` goes inside -> string goes outside, so no frontend is needed here.